src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -14,26 +14,55 @@
 
 package codeu.controller;
 
+import codeu.model.data.Activity;
+import codeu.model.store.basic.ActivityStore;
+import codeu.model.store.persistence.PersistentDataStoreException;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 
 public class ActivityFeedServlet extends HttpServlet {
 
+    private ActivityStore activityStore;
```

Comment:
Add the ActivityStore instance to the Servlet so that we can retrieve activities from DataStore